packages/app/lib/src/app/wrapper_page.dart

```diff
@@ -99,4 +103,94 @@ class WrapperPage extends ConsumerWidget {
       },
     );
   }
+
+  Future<void> _dialogBuilder(BuildContext context) {
```

Comment:
Extract it out to a private widget, i.e.:

```dart
class _DialogBuilder extends HookWidget {/*...*/}
```

Reply:
How would i implement this. I'm a little confused on what you mean here